src/internal/sdata/sdata_test.go

```diff
@@ -74,6 +80,21 @@ func TestMaterializeSQL(t *testing.T) {
 	}
 }
 
+// newSnowSQL creates an emphermeral database in a real Snowflake instance.
+func newSnowSQL(t testing.TB) *sqlx.DB {
+	user := os.Getenv("SNOW_USER")
```

Comment:
I'd prefer to write out "snowflake" so no one has to guess about what the environment variable is for if they see it outside this context.